assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.test.js

```diff
@@ -383,5 +398,86 @@ describe( 'AudienceSegmentationSetupCTAWidget', () => {
 
 			expect( queryByText( /Maybe later/i ) ).not.toBeInTheDocument();
 		} );
+
+		it( 'should call the enableAudienceGroups action when `Enable groups` CTA is clicked.', async () => {
+			const settings = {
+				configuredAudiences: [],
+				isAudienceSegmentationWidgetHidden: false,
+			};
+
+			enableAudienceGroupSpy.mockImplementation( jest.fn() );
+
+			// Set the data availability on page load to true.
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveIsDataAvailableOnLoad( true );
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveGetAudienceSettings( settings );
+
+			const { getByRole, queryByText } = render(
+				<AudienceSegmentationSetupCTAWidget Widget={ Widget } />,
+				{
+					registry,
+				}
+			);
+
+			expect(
+				getByRole( 'button', { name: /Enable groups/i } )
+			).toBeInTheDocument();
+
+			// eslint-disable-next-line require-await
+			await act( async () => {
+				fireEvent.click(
+					getByRole( 'button', { name: /Enable groups/i } )
+				);
+			} );
+
+			await waitFor( () =>
+				expect( enableAudienceGroupSpy ).toHaveBeenCalledTimes( 1 )
+			);
+			expect( queryByText( 'Enabling groups' ) ).toBeInTheDocument();
+		} );
+
+		it( 'should call the enableAudienceGroups action when `autoSubmit` is set to true.', async () => {
```

Comment:
Actually, the use of backticks is a little inconsistent here too. We might as well remove them from `autoSubmit`:

```suggestion
		it( 'should call the enableAudienceGroups action when autoSubmit is set to true.', async () => {
```